.editorconfig

```diff
@@ -195,5 +501,11 @@ end_of_line = crlf
 # IDE0040: Add accessibility modifiers
 dotnet_diagnostic.IDE0040.severity = warning
 
+# Rules for generated code for template engine
+[{*/obj/*/External/**/*,*/obj/dotnet-new.IntegrationTests/**}]
```

Comment:
what `*/obj/dotnet-new.IntegrationTests/**` stands for?

Reply:
It's mainly for the generated code file `TemplatePackagesPaths.cs` built from dotnet-new.IntegrationTests.

Reply:
Can we please be more specific here then.